stable/mlrun-kit/templates/minio-persistency/pvc.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.minioPersistencyStandalone.pv.create -}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: minio-pvc
+  labels:
+    app.kubernetes.io/name: {{ template "mlrun-kit.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+spec:
+  storageClassName: "minio-host-path"
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 2Gi
```

Comment:
same ^

Reply:
this is the pv that create the local volume